README.md

```diff
@@ -61,6 +61,14 @@ let serde_bytes = serde_json::to_vec(&desc)?;
 let borsh_bytes = borsh::to_vec(&desc)?;
 ```
 
+## Features
+
+| Feature | Default? | Description |
+| :--- | :---: | :--- |
+| `address` | ✓ | Adds descriptor support for Bitcoin addresses |
```

Comment:
I would like also to add ScriptBuf support to the description.

Reply:
Sorry to be picky but why not also monospace the "address" into a proper `Address`?

Reply:
I suppose we could. I was considering "Bitcoin address" as a more general term, and `ScriptBuf` as a specific struct. I'll update both so they link out for clarity.